<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Amazfit</class>
 <widget class="QMainWindow" name="Amazfit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="but_select_orig">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Select DB file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_generate">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Export GPX files</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_path_orig">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DB file</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Export Path</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_path_dest">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="but_select_dest">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Select export path</string>
    </property>
   </widget>
   <widget class="QLabel" name="endlabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
